addons/sale/sale.py

```diff
@@ -413,6 +414,18 @@ def _create_analytic_account(self, prefix=None):
             })
             order.project_id = analytic
 
+    @api.multi
+    def _get_taxes_by_group(self):
+        self.ensure_one()
+        res = {}
+        currency = self.currency_id or self.company_id.currency_id
+        for line in self.order_line:
```

Comment:
use self.mapped('order_line.tax_id').
for tax in self.mapped('order_line.tax_id')